actions/run.py

```diff
@@ -1,22 +1,33 @@
+'''
+ST2 AWS pack action runner script
+'''
+
 from lib import action
 
 
 class ActionManager(action.BaseAction):
+    '''
+    Main action class
+    '''
 
     def run(self, **kwargs):
-        action = kwargs['action']
+        '''
+        Action runner method
+        '''
+        aws_action = kwargs['action']
```

Comment:
Let's use `pop` here since the next line of code is to delete action and that can all be accomplished w/ a single call to `pop`.

Reply:
@bigmstone Thank you for your review! Sure thing, this could be "popped", I was just afraid to touch too much lines as first-time contributor here :smile: